nifi-nar-bundles/nifi-aws-bundle/nifi-aws-processors/src/main/java/org/apache/nifi/processors/aws/s3/CopyS3Object.java

```diff
@@ -69,11 +85,21 @@ public class CopyS3Object extends AbstractS3Processor {
             .defaultValue("${filename}-1")
             .build();
 
+    static final PropertyDescriptor MULTIPART_RETRIES = new PropertyDescriptor.Builder()
+            .name("Retry Attempt Limit")
+            .description("This configures the number of retries that will be attempted when a part upload request " +
+                    "on files larger than 5GB encounter a 503/Slow Down error.")
+            .defaultValue("3")
+            .addValidator(StandardValidators.POSITIVE_INTEGER_VALIDATOR)
+            .required(true)
+            .build();
```

Comment:
For this initial change, I think it is better to set a hard-coded retry of 3. Standard NiFi retry handling can support retrying the entire process, so having an internal retry attempt of 3 for individual part operations seems reasonable. This keeps the processor simpler from a user perspective.